soc/riscv/riscv-privileged/andes_v5/pma.c

```diff
@@ -82,45 +75,44 @@ static void write_pmaaddr_csr(const uint32_t index, unsigned long value)
 {
 	switch (index) {
 	case 0:
-		write_csr(NDS_PMAADDR0,  value); break;
+		csr_write(NDS_PMAADDR0,  value); break;
 	case 1:
-		write_csr(NDS_PMAADDR1,  value); break;
+		csr_write(NDS_PMAADDR1,  value); break;
 	case 2:
-		write_csr(NDS_PMAADDR2,  value); break;
+		csr_write(NDS_PMAADDR2,  value); break;
 	case 3:
-		write_csr(NDS_PMAADDR3,  value); break;
+		csr_write(NDS_PMAADDR3,  value); break;
 	case 4:
-		write_csr(NDS_PMAADDR4,  value); break;
+		csr_write(NDS_PMAADDR4,  value); break;
 	case 5:
-		write_csr(NDS_PMAADDR5,  value); break;
+		csr_write(NDS_PMAADDR5,  value); break;
 	case 6:
-		write_csr(NDS_PMAADDR6,  value); break;
+		csr_write(NDS_PMAADDR6,  value); break;
 	case 7:
-		write_csr(NDS_PMAADDR7,  value); break;
+		csr_write(NDS_PMAADDR7,  value); break;
 	case 8:
-		write_csr(NDS_PMAADDR8,  value); break;
+		csr_write(NDS_PMAADDR8,  value); break;
 	case 9:
-		write_csr(NDS_PMAADDR9,  value); break;
+		csr_write(NDS_PMAADDR9,  value); break;
 	case 10:
-		write_csr(NDS_PMAADDR10, value); break;
+		csr_write(NDS_PMAADDR10, value); break;
 	case 11:
-		write_csr(NDS_PMAADDR11, value); break;
+		csr_write(NDS_PMAADDR11, value); break;
 	case 12:
-		write_csr(NDS_PMAADDR12, value); break;
+		csr_write(NDS_PMAADDR12, value); break;
 	case 13:
-		write_csr(NDS_PMAADDR13, value); break;
+		csr_write(NDS_PMAADDR13, value); break;
 	case 14:
-		write_csr(NDS_PMAADDR14, value); break;
+		csr_write(NDS_PMAADDR14, value); break;
 	case 15:
-		write_csr(NDS_PMAADDR15, value); break;
+		csr_write(NDS_PMAADDR15, value); break;
```

Comment:
Looks like a lot of repeated code. Can you generate it with `FOR_EACH` helper macro?

Reply:
Nice suggestion, fixed it, thank you.